h/views/api/config.py

```diff
@@ -22,26 +24,42 @@
 
 
 def add_api_view(
-    config, view, link_name=None, description=None, enable_preflight=True, **settings
+    config,
+    view,
+    versions=None,
+    link_name=None,
+    description=None,
+    enable_preflight=True,
+    **settings
 ):
 
     """
     Add a view configuration for an API view.
 
-    This adds a new view using `config.add_view` with appropriate defaults for
-    API methods (JSON in & out, CORS support). Additionally if `link_name` is
-    specified it adds the view to the list of views returned by the `api.index`
-    route.
+    This configuration takes care of some common tasks for configuring
```

Comment:
I cleaned up this docstring but will do more better tightening later as I am going to be refactoring this function soon.